src/twisted/topfiles/8069.bugfix

```diff
@@ -0,0 +1 @@
+The transition to the hyperlink package adds IPv6 support to twisted.python.url.URL.
```

Comment:
It's deprecated but not removed.  That means interested users get a fix for [8069](http://twistedmatrix.com/trac/ticket/8069) even if they don't import `URL` from `hyperlink`.  I think this description should mention the deprecation ticket, though.